test/e2e/network/topology_hints.go

```diff
@@ -83,6 +84,33 @@ var _ = common.SIGDescribe("Feature:Topology Hints", func() {
 		framework.ExpectNoError(err, "timed out waiting for DaemonSets to be ready")
 
 		nodeNames := e2edaemonset.SchedulableNodes(c, ds)
+
+		// All Nodes should have same allocatable CPUs. If not, then skip the test.
+		nodeNamesMap := map[string]bool{}
```

Comment:
Maybe a slight rename to indicate that this was specific to schedulable Nodes? Not tied to this specific name, but would help make the rest of the code a bit easier to understand. 
```suggestion
		schedulableNodes := map[string]bool{}
```

Reply:
done